### PR TITLE
allow to add to calibration_devices

### PR DESCRIPTION
**Pull request by tobiaskleiner**
This PR allows to add json schemas to the collection of calibration_devices. I have added a TODO to modify the code later to be more general and to remove the naming of telescope and adjust the validation methods.

### CODE REVIEW

File: simtools/db/db_handler.py [hunk 1]
```diff
@@ -1039,8 +1039,14 @@ def add_new_parameter(
             db_entry["instrument"] = names.validate_telescope_name(telescope)
         elif "sites" in collection_name:
             db_entry["instrument"] = names.validate_site_name(site)
+        # TODO: generalize later
```
Comment (GernotMaier) [inline simtools/db/db_handler.py:1042]: Suggest to remove this comment and open an issue regarding the generalization.

File: simtools/db/db_handler.py [hunk 2]
```diff
@@ -1039,8 +1039,14 @@ def add_new_parameter(
...
+        elif "calibration_devices" in collection_name:
```
Comment (GernotMaier) [inline simtools/db/db_handler.py:1043]: Do you want to add one single test for this in test_adding_new_parameter_db in test_db_handler? A single line is probably enough.

**Review by GernotMaier — APPROVED** [review]
Two minor comments - but fine as it is here.

File: simtools/db/db_handler.py (hunk 1 above)
Comment (tobiaskleiner) [reply simtools/db/db_handler.py]: Removed, opening an issue.

File: simtools/db/db_handler.py (hunk 2 above)
Comment (tobiaskleiner) [reply simtools/db/db_handler.py:1042]: Added.

**Review by GernotMaier — APPROVED** [review]
Approved (again - all good!)

File: simtools/db/db_handler.py (hunk 1 above)
Comment (tobiaskleiner) [reply simtools/db/db_handler.py]: Issue is here: #871

**Pull request merged**